src/ddmd/toir.d

```diff
@@ -348,6 +348,48 @@ elem *setEthis(Loc loc, IRState *irs, elem *ey, AggregateDeclaration ad)
  */
 int intrinsic_op(FuncDeclaration fd)
 {
+    // simplified calculation of back references for core.simd symbols
```

Comment:
> More explanation would be nice (and why it is never executed?).

It is executed at compile time only when initializing core_namearray or core_namearray64. I used to version this out to switch between the old and the new mangling for debugging.

Maybe it's just better to replace the string literals with their new mangled names...